src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/set_interior_vehicle_data_request_test.cc

```diff
@@ -186,38 +188,36 @@ TEST_F(SetInteriorVehicleDataRequestTest,
 TEST_F(
     SetInteriorVehicleDataRequestTest,
     Execute_ValidWithSettableAndReadOnlyParams_ExpectCutReadOnlyAndResendToHMI) {
-  // Arrange
+  using namespace rc_rpc_plugin::message_params;
 
-  MessageSharedPtr mobile_message = CreateBasicMessage();
-  ns_smart_device_link::ns_smart_objects::SmartObject& msg_params =
+  auto mobile_message = CreateBasicMessage();
+  auto& msg_params =
       (*mobile_message)[application_manager::strings::msg_params];
-  msg_params[message_params::kModuleData][message_params::kModuleType] =
-      mobile_apis::ModuleType::RADIO;
-  smart_objects::SmartObject radio_control_data(smart_objects::SmartType_Map);
-  radio_control_data[message_params::kState] = true;
-  radio_control_data[message_params::kRadioEnable] = true;
-  msg_params[message_params::kModuleData][message_params::kRadioControlData] =
-      radio_control_data;
+  msg_params[kModuleData][kModuleType] = mobile_apis::ModuleType::RADIO;
+  auto& control_data = msg_params[kModuleData][kRadioControlData];
+  control_data[kState] = true;
+  control_data[kRadioEnable] = true;
 
-  // Expectations
```

Comment:
@mvorobio What is the reason to remove such comments?

Reply:
@AByzhynar please check https://github.com/mvorobio/sdl_core/pull/2/commits/794597b6469711802915fb788ab9bb4fec9ae004